SCClassLibrary/Common/Math/Complex.sc

```diff
@@ -95,6 +95,20 @@ Complex : Number {
 		^Complex(real * denom, imag.neg * denom)
 	}
 
+	sqrt {
+		var mag, sign;
+
+		mag = this.magnitude;
+		sign = (imag.sign + 1).asBoolean.if({ 1 }, { 1.neg });  // +1 >= 0, -1 < 0
```

Comment:
I would rewrite this as:
```
sign = if(imag.isPositive) { 1 } { -1 }
```

Reply:
i had to look it up -- `isPositive` is more like `isNonNegative`, while `isStrictlyPositive` actually tests `n > 0`.

Reply:
too bad we don't have isNonNegative for Numbers :(